lua/nvim-treesitter/parsers.lua

```diff
@@ -325,6 +325,15 @@ list.cpp = {
   maintainers = { "@theHamsta" },
 }
 
+list.crystal = {
+  install_info = {
+    url = "https://github.com/crystal-lang-tools/tree-sitter-crystal",
+    files = { "src/parser.c", "src/scanner.c", "src/unicode.c" },
```

Comment:
```suggestion
    files = { "src/parser.c", "src/scanner.c", "src/unicode.c" },
```

That is not really portable. Can you test that `tree-sitter build -o crystal.so` works correctly?

Reply:
@hugopl `unicode.c` doesn't need to be in this list, `scanner.c` actually just includes it inline


@clason 
The "Compile parsers" step on Windows includes these logs:
```
Downloading tree-sitter-crystal...
Creating temporary directory
Extracting tree-sitter-crystal...
Compiling...
Treesitter parser for crystal has been installed
```
Do you think that's failing silently?

Reply:
Yes, since it then complains about the parser not being installed.

Reply:
Fixed.